pkg/api/norman/customization/vsphere/listers.go

```diff
@@ -51,6 +52,37 @@ func processSoapFinder(ctx context.Context, fieldName string, cc *v1.Secret, dc
 	return data, err
 }
 
+func processSoapFinderExtended(ctx context.Context, fieldName string, cc *v1.Secret, dc string) ([]map[string]interface{}, error) {
+	finder, err := getSoapFinder(ctx, cc, dc)
+	if err != nil {
+		return nil, err
+	}
+	var data []map[string]interface{}
+	switch fieldName {
+	case "networks-extended":
+		data, err = listNetworksExtended(ctx, finder)
+	}
+	return data, nil
+}
+
+func listNetworksExtended(ctx context.Context, finder *find.Finder) ([]map[string]interface{}, error) {
+	networks, err := finder.NetworkList(ctx, "*")
+	if err != nil {
+		return nil, err
```

Comment:
Is there any context we can provide for this error?

My reading of the code is that you will get a fairly generic error back https://github.com/vmware/govmomi/blob/d4dc69963226c4cba8dd5a7f5289bf76163fac7f/find/finder.go#L792 ?

Reply:
Errors are wrapped with some meaningful information now.